randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -1394,7 +1394,7 @@ Extra - minimap_data: [{'x': 66, 'y': 49}, {'x': 66, 'y': 50}, {'x': 66, 'y': 51
 
 > Door to Serris Arena; Heals? False
   * Layers: default
-  * Missile Door to Serris Arena/Door to Serris Arena Access; Dock Lock Rando incompatible with: Spider Ball Door, Screw Attack Door
+  * Missile Door to Serris Arena/Door to Serris Arena Access; Dock Lock Rando incompatible with: Spider Ball Door, Screw Attack Door, Hydro Station Water Turbine
```

Comment:
Spider Ball exclusion on all these doors? for what reason? iirc, you can use Spider underwater.

Reply:
Spider doesnt work underwater without gravity. It only works in the emp area without gravity